### PR TITLE
Adding the methodology section

### LINKED ISSUE #67: ReadMe: Add the Methodology Section
Can we please have an expansion of the Methodology section in the ReadMe? Make sure you include these:

- [My Experience on Spotify Agile Methodology](https://blog.praveen.science/my-experience-on-spotify-agile-methodology/)
- [Git Standards followed in our way of Spotify Agile Methodology](https://blog.praveen.science/git-standards-followed-in-our-way-of-spotify-agile-methodolgy/)

We're using **Spotify Agile Methodology**. Let's discuss what you're planning to write before you're being assigned! 😁

Up for grabs! ✋🏻

### PR DESCRIPTION
**Pull request by vismitap**
Fixed #67

## Related Issue 
Closes: #67 

### Describe the changes you've made
Added the methodology section in the README.md file.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

Describe how have you verified the changes made

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly whereever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots
image

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -41,6 +41,15 @@ If you're interested, why not start with:
 
 ### Methodology
 
+We are following **Spotify Agile Methodology** to give the contributors an idea of efficient software development methods and how work is co-ordinated in the industry. If you don't know what that means, please refer [this blog](https://blog.praveen.science/my-experience-on-spotify-agile-methodology/). 
+Following are some points on the methodology to be used:
```
Comment (praveenscience) [inline ReadMe.md:45]: I'll add a space above and below this line to make it a paragraph.

File: ReadMe.md [hunk 2]
```diff
@@ -41,6 +41,15 @@ If you're interested, why not start with:
...
+- To get the best out of everyone contributing to this project, we have adopted high alignment and high autonomy model.
+- The issues created will have a description of the problem to be solved, the contributors have to figure out how they would like to solve it and what tool they would like to use.
+- Everything needs to be communicated to the scrum masters/project admin on what they think about a particular issue and its possible solution.
+- We will have a sprint of 3days-5days based on the level of difficulty of the issue with regular sprint planning sessions and daily stand-ups to track the progress of each and every contributor.
+- In these sessions, the scrum masters will take a note of the progress and blockers if any. If the contributor faces any issues/blocks, the others will help them solve it and resume their work. Hence, the blocks can be addressed timely. 
+- To know more about the Git standards we will be following, refer [this](https://blog.praveen.science/git-standards-followed-in-our-way-of-spotify-agile-methodolgy/).
```
Comment (praveenscience) [inline ReadMe.md:51]: Better linking strategy can be used.

**Review by praveenscience — APPROVED** [review]
@vismitap This is the reason, you're awesome! ❤️ I'll do the rest of the changes and get this merged. Thanks! 😊

**Comment by vismitap** [comment]
> @vismitap This is the reason, you're awesome! ❤️ I'll do the rest of the changes and get this merged. Thanks! 😊

Thanks! 👍😄

**Pull request merged**